image/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6.9-slim-buster
+ 
+ENV DEBIAN_FRONTEND noninteractive 
+ 
+RUN apt-get update && apt-get install -y --no-install-recommends --no-install-suggests \ 
+  git \
+  vim
+  libcurl4-openssl-dev
+
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+
+RUN echo "deb http://http.debian.net/debian buster main" > /etc/apt/sources.list.d/debian-unstable.list
+
+RUN pip install pronto pytest jupyter rpy2 biopython flask click
+RUN pip install --force-reinstall git+'https://github.com/HUPO-PSI/mzQC@mzqc-pylib#egg=MZQC-0.0.1&subdirectory=pylib'
```

Comment:
it doesn't build for me. I got the following error:

```
Step 7 : RUN pip install pronto pytest jupyter rpy2 biopython flask click
 ---> Running in 09c737dcf7e9
Collecting pronto
  Downloading https://files.pythonhosted.org/packages/01/4e/525b8fda6ed9ce8e49b9db464a13132ca82aefec5431f0f5797c521ff18f/pronto-1.1.3-py2.py3-none-any.whl (47kB)
Collecting pytest
  Downloading https://files.pythonhosted.org/packages/93/16/f6dec5178f5f4141e80dfc4812a9aba88f5f29ca881f174ab1851181d016/pytest-5.2.2-py3-none-any.whl (227kB)
Collecting jupyter
  Downloading https://files.pythonhosted.org/packages/83/df/0f5dd132200728a86190397e1ea87cd76244e42d39ec5e88efd25b2abd7e/jupyter-1.0.0-py2.py3-none-any.whl
Collecting rpy2
  Downloading https://files.pythonhosted.org/packages/d9/51/d5c41ac70e0aa3272f24cf836bd35d6246c95b98d8d811f338cec5a8098e/rpy2-3.2.2.tar.gz (162kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-k_95fld0/rpy2/setup.py'"'"'; __file__='"'"'/tmp/pip-install-k_95fld0/rpy2/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-k_95fld0/rpy2/pip-egg-info
         cwd: /tmp/pip-install-k_95fld0/rpy2/
    Complete output (1 lines):
    There is no R_HOME and the R executable cannot be found.
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
The command '/bin/sh -c pip install pronto pytest jupyter rpy2 biopython flask click' returned a non-zero code: 1
```

Reply:
Do we really need R?
```
    There is no R_HOME and the R executable cannot be found.
`
``

Reply:
Yes, for the basic stuff in the conversion, we can remove the rpy2.